corehq/apps/geospatial/views.py

```diff
@@ -90,22 +91,22 @@ def page_url(self):
 
     @property
     def page_context(self):
-        config = self.config
-
         return {
             'form': self.settings_form,
-            'config': {
-                'location_data_source': config.location_data_source,
-                'user_location_property_name': config.user_location_property_name,
-                'case_location_property_name': config.case_location_property_name,
-            }
+            'config': model_to_dict(
+                self.config,
+                fields=[
+                    'location_data_source',
+                    'user_location_property_name',
+                    'case_location_property_name',
+                ])
```

Comment:
If you want to DRY this even more, you could go with
```suggestion
                fields=GeospatialConfigForm.Meta.fields)
```

Reply:
Oooh, good suggestion!